pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
 dynamic = ["version"]
 
 [tool.setuptools]
-packages = ["emu_base", "emu_mps", "emu_sv", "optimatrix"]
+packages = ["emu_base", "emu_mps", "emu_sv", "emu_mps.optimatrix"]
```

Comment:
I'm not sure we want to release this as a separate package then?

Reply:
removed